docs/GHA.md

```diff
@@ -0,0 +1,50 @@
+# GitHub Actions
+
+The Buildkite Migration tool supports transitioning GitHub Action workflow definitions to Buildkite pipelines.
+
+## Partially supported properties
+
+### concurrency
+
+The `concurrency` key within a GitHub Action workflow can be defined at topmost level, and within a single job. The Buildkite Migration tool supports the transition of the `concurrency` key (and `group`/`cancel-in-progress`) in job definitions: and maps to the `concurrency_group` [key](https://buildkite.com/docs/pipelines/controlling-concurrency#concurrency-groups) available within Buildkite. Buildkite also allows a upper limit on how much jobs are created through a single step definition with the `concurrency` key: which is set as `1` by default (there isn't a translatable key within a GitHub Action workflow).
+
+The `cancel-in-progress` Boolean value that can be defined in a `concurrency` hash inside a GitHub Action job workflow maps to the Buildkite pipeline setting of [Cancel Intermediate Builds](https://buildkite.com/docs/pipelines/skipping#cancel-running-intermediate-builds) - which can be set within a pipeline's settings page, or when creating/updating a pipeline via the [REST](https://buildkite.com/docs/apis/rest-api/pipelines#create-a-yaml-pipeline)/[GraphQL](https://buildkite.com/docs/apis/graphql/schemas/mutation/pipelinecreate) APIs.
+
+### env 
+
+GitHub Actions allows the specification of environment variables at the workflow level, at a `job` level, and for each `step` within a `job`.
+
+Environment variables that are defined at the top of a workflow will be transition to [build level](https://buildkite.com/docs/pipelines/environment-variables#environment-variable-precedence) environment variables in the generated Buildkite pipeline. Environment variables defined within the context of each of a workflows' `jobs` are transitioned to [step level](https://buildkite.com/docs/pipelines/environment-variables#runtime-variable-interpolation) environment variables.
+
+### jobs
+
+GitHub Action workflows allows you to specify one or more `jobs` - main tasks of a workflow that run in parallel by default.
+
+The Buildkite Migration tool currently supports the following 
+
+- `runs_on`: The `runs_on` key defines the type of machine that the job will run on. Within Buildkite, this is mapped to an agent targeting [tag](https://buildkite.com/docs/agent/v3/queues#targeting-a-queue) of `runs_on`. Note that jobs that target custom `tag` names will have a `queue` target of `default`.
+- `steps`: Steps that are defined for a particular `job`. Any `run` key is supported - and at present, any action that defines a `uses` attribute currently is not supported.
+
+## Unsupported properties
```

Comment:
I would split these in 2 sections, those that are not supported because of lack of functionality (defaults) and those that are have no translation (name, on, permissions)

Reply:
Split